doc/modules/svm.rst

```diff
@@ -392,6 +392,11 @@ Tips on Practical Use
   * **Setting C**: ``C`` is ``1`` by default and it's a reasonable default
     choice.  If you have a lot of noisy observations you should decrease it.
     It corresponds to regularize more the estimation.
+    
+    :class:`LinearSVC` and :class`LinearSVR` are less sensitive to ``C`` when
+    it becomes large, and prediction results stop improving after a certain 
+    threshold. In fact, large ``C`` values will take more time to train, 
```

Comment:
I find "in fact" strange here. I think "however" or "meanwhile" or "at the same time" are more appropriate